distros/kubernetes/nvsentinel/charts/preflight/templates/clusterrole.yaml

```
@@ -0,0 +1,49 @@
+  # Volcano PodGroups for gang discovery 
+  # - apiGroups: ["scheduling.volcano.sh"]
+  #   resources: ["podgroups"]
+  #   verbs: ["get", "list", "watch"]
+  
+  # OSMO/KAI PodGroups for gang discovery (custom scheduler)
+  # - apiGroups: ["scheduling.run.ai"]
+  #   resources: ["podgroups"]
+  #   verbs: ["get", "list", "watch"]
+  
+{{- end }}
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (1 > 0)

(empty-lines)